docs/source/interfaces/direct/Crystal Field Python Interface.rst

```diff
@@ -156,13 +156,17 @@ The three peaks now have all different widths. The first peak (index 0) keeps th
    :height: 300
 
 If called without arguments `getSpectrum()` determines automatically the range and number of the `x`-points. To have more control
-of how the spectrum is calculated a list (or numpy array) of x-values can be provided as a first argument to `getSpectrum`.
+of how the spectrum is calculated either a list (or numpy array) of x-values or a tuple with the range for the x values can be
+provided as a first argument to `getSpectrum`.
```

Comment:
Should this say that the x range can be provided as a keyword argument using `x_range`, rather than saying "provided as a first argument"? "Provided as a first argument" would suggest to me that I could provide an x range as a first argument without using the `x_range` keyword